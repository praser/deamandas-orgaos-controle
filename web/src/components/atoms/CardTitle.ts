import styled from 'styled-components';

const CardTitle = styled.h6`
  /* Text */
  font-family: Nunito, -apple-system, system-ui, 'Segoe UI', Roboto,
    'Helvetica Neue', Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji',
    'Segoe UI Symbol', 'Noto Color Emoji';
  font-size: 16px;
  font-weight: 700;
  line-height: 19.2px;
  text-align: left;
  text-size-adjust: 100%;

  /* Appearance */
  color: rgb(78, 115, 223);
`;

export default CardTitle;
