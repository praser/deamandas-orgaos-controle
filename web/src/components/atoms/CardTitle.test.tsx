import React from 'react';
import { render, screen } from '@testing-library/react';
import CardTitle from './CardTitle';

describe('CardTitle component', () => {
  let cardtitle: HTMLElement;
  const testId = 'testid';

  beforeEach(() => {
    render(<CardTitle data-testid={testId} />);
    cardtitle = screen.getByTestId(testId);
  });

  it('is expected to match the snapshot', () => {
    expect(cardtitle).toMatchSnapshot();
  });
});
