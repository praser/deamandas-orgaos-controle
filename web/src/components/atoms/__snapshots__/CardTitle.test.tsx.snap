// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardTitle component is expected to match the snapshot 1`] = `
<h6
  class="sc-bdvvaa lkGnPW"
  data-testid="testid"
/>
`;
